marketdata/stream/message.go

```diff
@@ -626,6 +724,7 @@ var subMessageHandler = func(c *client, s subscriptions) error {
 	c.sub.lulds = s.lulds
 	c.sub.cancelErrors = s.cancelErrors
 	c.sub.corrections = s.corrections
+	c.sub.news = s.news
```

Comment:
I think you may have to make the corresponding change in  `handleSubscriptionMessage`.